tests/Drupal/InfoFiles/DependencyArraySniffUnitTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Wrong file name, should be DependencyArrayUnitTest.php

Reply:
Thank you for correcting me. I will fix this.